src/atoms/title/AppTitle.ts

```diff
@@ -0,0 +1,29 @@
+import { TTitle } from '../../types';
+
+export function AppTitle({
+    tag = 'h1',
+    textContent = '',
+    innerHTML,
+    classes = [],
+}: TTitle): HTMLElement {
+    const headerTags: (keyof HTMLElementTagNameMap)[] = [
+        'h1',
+        'h2',
+        'h3',
+        'h4',
+        'h5',
+        'h6',
+    ];
+    const headerTag = headerTags.includes(tag) ? tag : 'h1';
+    const title = document.createElement(headerTag);
+
+    if (innerHTML) {
+        title.innerHTML = innerHTML;
```

Comment:
Please use `innerHTML` with caution, what would be the use case of it for a title?